ops/op2/dispatch_fast.rs

```diff
@@ -503,7 +518,7 @@ pub(crate) fn generate_dispatch_fast(
 
     #[allow(clippy::too_many_arguments)]
     extern "C" fn #fast_function(
-      _: deno_core::v8::Local<deno_core::v8::Object>,
```

Comment:
Let's split this one line change into its own PR + the test changes to reduce the PR size.

Reply:
it will be a warning (unused argument) if landed seperately

Reply:
No worries -- just add `allow_unused` for it. We can remove that at a later date.